dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -365,6 +365,16 @@
       <ThemeResource>true</ThemeResource>
       <Link>Themes\%(Filename)%(Extension)</Link>
     </Page>
+    <Page Include="$(MSBuildProjectDirectory)\Density\Compact.xaml">
+      <SubType>Designer</SubType>
```

Comment:
This file is in the wrong location in the PRI file to work the way you want via Nuget consumption. In Pri the resource is here:

     <NamedResource name="Compact.xbf" uri="ms-resource://Microsoft.UI.Xaml/Files/Microsoft.UI.Xaml/Density/Compact.xbf">
      <Candidate type="EmbeddedData">
       <Base64Value>[Embedded Binary Data]</Base64Value>
      </Candidate>
     </NamedResource>

Also this change doesn't have the work to make this work in framework package. I think we should just do that now and test it instead of leaving it for later.

Reply:
Oh wait, I see now that's the path you wanted it in. Ok. Then the remaining issue is to make this work in the framework package flavor of the nuget package.

Reply:
framework package works now with         
`<ResourceDictionary Source="ms-appx:///Microsoft.UI.Xaml/DensityStyles/Compact.xaml" />`